packages/osd-plugin-generator/template/public/components/app.tsx.ejs

```diff
@@ -88,10 +88,10 @@ import { CoreStart } from '<%= importFromRoot('src/core/public') %>';
                                           defaultMessage="Last timestamp: {time}" values={{ time: timestamp ? timestamp
                                           : 'Unknown' }} />
                                       </p>
-                                      <EuiButton type="primary" size="s" onClick={onClickHandler}>
+                                      <EuiSmallButton type="primary" size="s" onClick={onClickHandler}>
                                         <FormattedMessage id="<%= camelCase(name) %>.buttonText"
                                           defaultMessage="<%= hasServer ? 'Get data' : 'Click me' %>" />
-                                      </EuiButton>
+                                      </EuiSmallButton>
```

Comment:
can we fix the import by change `EuiSmallButton` to `EuiButton`, so the page style get unchanged.

Reply:
yes sure i will do that change